docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -83,19 +80,21 @@ Use this to orient the ADC to determine the location to insert your wires.
 {{% /alert %}}
 
 Insert the MCP3008 into your breadboard so that it bridges both sides of the divide.
-Then, use the rows on the side of your MCP3008's pins and the GPIO pins on your PI to connect the pins with wires as follows:
+Then, use the rows on the side of your MCP3008's pins and the GPIO pins on your Pi to connect the pins with wires as follows:
 
 <!-- prettier-ignore -->
 | MCP3008 ADC Pin | Raspberry Pi Pin |
 | ----------- | ---------------- |
-| VDD | 3.3V |
-| VREF | 3.3V |
+| VDD | 5V |
+| VREF | 5V |
 | AGND | GND |
 | DGND | GND |
 | CLK | SCLK |
```

Comment:
This is outside of the diff so feel free to disregard but, should these say which pin number they are?

Reply:
I want to believe this was an intentional choice at the beginning because the adc does work across boards if you just use the chip select, miso, clock, mosi, power, ground etc. pins on the board and the pin number is not that important (more so the pin purpose). And some of these go to the breadboard rail and not to actual power pins on the Pi. But I see where you are coming from, if you think it makes it easier for the reader to just see the pins, I can add that in the next iteration

Reply:
Okay that makes sense! Can disregard my comment here!